### DISCUSSION TITLE
inconsistent frontend

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#7948] [Q&A]
It occurs to me that there are multiple frontend co-existing now and sometimes I'll get tumble one and only if I use incognitive tab it will get me to the latest one, is this expected?

**Comment by Rezanmonis** [comment]
We are looking for a frontend developer as our teammate for our edtech project rezanmonis@gmail.com

**Comment by Shinusagi** [comment]
I just realized there are few issues with the new frontend so far:

It gives an error when creating/editing some Part/Stock because the verification check of "Allow this change" is not being shown and I assume it is disable by default usually, so it will prevent the registry creation/change.

There is no option to see which Parts you are subscribed to, and thus organize your notifications over them.

The first one is a really annoynig error TBH.

**Comment by SchrodingersGat** [comment]
> It gives an error when creating/editing some Part/Stock because the verification check of "Allow this change" is not being shown and I assume it is disable by default usually, so it will prevent the registry creation/change.

What do you mean by "allow this change"?

> There is no option to see which Parts you are subscribed to, and thus organize your notifications over them.

Good point, I'll raise an issue so we can track this

**Comment by Shinusagi** [comment]
In the old interface if I try to change the count on something I see this:

![imagen](https://github.com/user-attachments/assets/618daa3c-80d9-4cc8-b1cc-be1ab9ba05a1)

As you can see there is the "Confirm stock adjustment" option which until enabled won't let save the changes, once you do that everything works fine... however in the new interface:

![imagen](https://github.com/user-attachments/assets/0e9f2447-3a24-4213-9006-0403a8aca437)

There is no option and you can click directly on "Submit" but if you do you get this error:
![imagen](https://github.com/user-attachments/assets/4ec92fb4-da2d-4d48-9558-0b69df309921)

I've seen this issue even creating new parts or stuff, so I thought it was because this setting:

![imagen](https://github.com/user-attachments/assets/133cedbf-7cce-4200-a118-1e789930699f)

Then tried again and I don't need confirmation on the Old Interface but in the New Interface I still get an error so maybe it is unrelated, not sure what is the issue then but that is the reason I am not using the new interface, sorry.

**Comment by SchrodingersGat** [comment]
What version are you running?  The error in the "count stock" form should be fixed in the most recent version.  It is not related to the "confirm" button

**Reply by Shinusagi** [reply]
![imagen](https://github.com/user-attachments/assets/edfa66e7-f258-4768-a35f-00d39b9e07d9)
0.16.1
